primitives/header-chain/src/justification.rs

```diff
@@ -59,7 +59,7 @@ pub fn verify_justification<Header: HeaderT>(
 	authorities_set_id: SetId,
 	authorities_set: &VoterSet<AuthorityId>,
 	raw_justification: &[u8],
-) -> Result<(), Error>
+) -> Result<(usize, usize), Error>
```

Comment:
Why not return a `GrandpaJustification` instead and let the caller read values? Two `usize` variables might be confusing.